### PR TITLE
SNOW-802269 To Add reverse isnull conv unixtimestamp

### PR DESCRIPTION
**Pull request by sfc-gh-sjayabalan**
Please answer these questions before submitting your pull requests. Thanks!

1. What GitHub issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   Added following functionalities
    Reverse function
    isnull
    conv
    unixtimestamp

2. Fill out the following pre-review checklist:

   - [ yes] I am adding a new automated test(s) to verify correctness of my new code
   - [ ] I am adding new logging messages
   - [ ] I am adding a new telemetry message
   - [ ] I am adding new credentials
   - [ ] I am adding a new dependency

3. Please describe how your code solves the related issue.

   *Replace this with a short description of how your code change solves the related issue.*

## Pre-review checklist

(For Snowflake employees)

- [ ] This change has passed precommit
- [ ] I have reviewed code coverage report for my PR in  ([Sonarqube](https://sonarqube.int.snowflakecomputing.com/dashboard?id=snowpark))

### CODE REVIEW

File: src/main/java/com/snowflake/snowpark_java/Functions.java [hunk 1]
```diff
@@ -3880,6 +3881,75 @@ public static Column listagg(Column col, String delimiter) {
...
+   * |-------------------------+------------------------|
+   * | 2013-05-08 23:39:20.123 |             1368056360 |
+   * +-------------------------+------------------------+
+   * }</pre>
+   *
+   * @since 1.14.0
+   * @param C Column to be converted.
+   * @return Column object.
+   */
+  public static Column unix_timestamp(Column C) {
```
Comment (sfc-gh-bli) [inline src/main/java/com/snowflake/snowpark_java/Functions.java:3950]: why capital `C`?

File: src/main/java/com/snowflake/snowpark_java/Functions.java [hunk 2]
```diff
@@ -8,6 +8,7 @@
 import com.snowflake.snowpark_java.udf.*;
 import java.util.List;
 import java.util.function.Supplier;
+import net.snowflake.client.jdbc.internal.org.checkerframework.checker.units.qual.C;
```
Comment (sfc-gh-bli) [inline src/main/java/com/snowflake/snowpark_java/Functions.java:11]: What is it?

File: src/test/java/com/snowflake/snowpark_test/JavaFunctionSuite.java [hunk 3]
```diff
@@ -2766,6 +2766,27 @@ public void any_value() {
...
+  }
+
+  @Test
+  public void isnull() {
+    DataFrame df = getSession().sql("select * from values(1.2),(null),(2.3) as T(a)");
+    Row[] expected = {Row.create(false), Row.create(true), Row.create(false)};
+    checkAnswer(df.select(Functions.isnull(df.col("a"))), expected, false);
+  }
+
+  public void unix_timestamp() {
```
Comment (sfc-gh-bli) [inline src/test/java/com/snowflake/snowpark_test/JavaFunctionSuite.java:2781]: `@Test` ?

File: src/main/java/com/snowflake/snowpark_java/Functions.java (hunk 1 above)
Comment (sfc-gh-sjayabalan) [reply src/main/java/com/snowflake/snowpark_java/Functions.java]: modified

**Review by sfc-gh-bli — APPROVED** [review]
LGTM

**Pull request merged**